_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -0,0 +1,169 @@
+
+<!-- more -->
+
+<center>
+ <img src="/images/tech-2017/businesses.svg" style="width:100%;">
+</center>
+
+
+# Organizational Structures
+
+In 2016, we [updated the Engineering organization](/blog/2016/03/28/artsy-engineering-organization-stack) to be oriented around product verticals. Since then, web and mobile "practices" have largely been subsumed into these separate product teams. Mobile's increasing reliance on React Native has aligned nicely with web tooling. It no longer made sense to keep the teams separate, so where product teams used to have 2 separate sub-teams of engineers, they've now merged into 1.
+
+The Platform "practice" has remained as a way to coordinate and share work among product teams, as well as monitor and upgrade Artsy's platform over time. Most platform engineers operate from within product teams, while a few focusing on data and infrastructure form a core, dedicated Platform team.
+
+<center>
+ <img src="/images/tech-2017/engineering-teams.svg" style="width:100%;">
+</center>
+
+# Artsy Technology Infrastructure
+
+<center>
+ <img src="/images/tech-2017/languages.svg" style="width:100%;">
+</center>
+
+
+## User Facing
+
+A lot of the user-facing focus includes being able to present interfaces with a quality worthy of art.
+
+What you see today when you go to [www.artsy.net](https://artsy.net) is a website built with [Ezel.js](http://ezeljs.com), which is a boilerplate for [Backbone](http://backbonejs.org) projects running on [Node](https://nodejs.org) and using [Express](http://expressjs.com) and [Browserify](http://browserify.org). We used to have separate projects for mobile and desktop web, but they [are now merged](https://github.com/artsy/force/pull/890). The combined app is hosted on [Heroku](http://heroku.com) and uses [Redis](http://redis.io) for caching. Assets, including artwork images, are served from [Amazon S3](http://aws.amazon.com/s3) via the [CloudFront CDN](http://aws.amazon.com/cloudfront). This [code is open-source](https://github.com/artsy/force).
+
+What you see today when you open the [Artsy iOS app](https://itunes.apple.com/us/app/artsy-collect-and-bid-on-fine-art-design/id703796080?mt=8) is a mix of Objective-C, Swift and React Native. Objective-C and Swift continue to provide a lot of over-arching cross-View Controller code. While individual representations of Artsy resources tend to be built in React Native. All of our React Native code uses Relay to handle API integration. This [code is open-source](https://github.com/artsy/eigen).
+
+You can also find Artsy on [Alexa](http://alexa.atsy.net) and [Google Home](http://assistant.artsy.net), which are both open-source Node.js applications.
```

Comment:
Do we also want to add Apple TV's shows app? (seeing as it's kinda "home" as well?)